tests/unit/backend/layers/business/test_business.py

```diff
@@ -343,17 +379,21 @@ def test_update_collection_wrong_params_fail(self):
             {"name": "contact_email", "reason": "Cannot be blank."},
         ])
 
-        # TODO: I would recommend adding more validation logic testing (see existing TestVerifyCollection)
-        # in a separate unit test case later
-
     def test_update_published_collection_fail(self):
         """
         Updating a collection version that is published should fail
         """
         version = self.initialize_published_collection()
+        body = CollectionMetadataUpdate(
+            name="new collection name",
+            description="new collection description",
+            contact_name=None,
+            contact_email=None,
+            links=None,
+        )
```

Comment:
```suggestion
        body = CollectionMetadataUpdate(
            name="new collection name",
            description="new collection description",
            contact_name="new contact name",
            contact_email="new_email@czi.com",
        )
```

Reply:
Otherwise you are testing 'missing parameters' at the same time as 'update published'